web-components/src/components/inputs/new/LocationPicker/LocationPicker.tsx

```diff
@@ -129,7 +129,7 @@ export const LocationPicker = ({
         </div>
         {/* We simply absolutely position the location pin at the center,
           without using a react-map-gl Marker, so the map moves around the pin */}
-        <div className="absolute top-[50%] left-[50%]">
+        <div className="absolute top-[50%] left-[50%] -ml-[18.5px] -mt-50">
```

Comment:
18.5px corresponds to half the width of the pin icon and 50px its height, we shift the pin icon a bit so that the bottom middle part of it is on the map center